index.js

```diff
@@ -6,12 +6,16 @@
  * @param  {Boolean} options.hasHeader - this will determine if the table data has a header or not, by default this is true
  * @param  {String}  options.columnSeparator - the default separators is ' | '
  * @param  {String}  options.headerSeparator - the default header separators is '=', this value should only be one character
- * @return {String} - a string that represents the ascii table of the data provided
+ * @return {Object} - { ascii - a string that represents the ascii table of the data provided,
+ *                      html - a provided html version of table,
+ *                      table - html <table> table,
+ *                      markdown - a markdown table }
  */
 module.exports = function turtler(data, options={}) {
```

Comment:
what we should do is make the main file return a turtler class, so that we can extend the object.